lib/codecs/src/decoding/format/syslog.rs

```diff
@@ -101,14 +110,88 @@ impl SyslogDeserializerConfig {
                 // "structured data" is placed at the root. It will always be a map strings
                 .unknown_fields(Kind::object(Collection::from_unknown(Kind::bytes())))
             }
+            (LogNamespace::Vector, Some(source)) => {
+                schema::Definition::new_with_default_metadata(Kind::bytes(), [log_namespace])
+                    .with_meaning(LookupBuf::root(), "message")
+                    .with_source_metadata(
+                        source,
+                        None,
+                        &owned_value_path!("timestamp"),
+                        Kind::timestamp(),
+                        Some("timestamp"),
+                    )
+                    .with_source_metadata(
+                        source,
+                        None,
+                        &owned_value_path!("hostname"),
+                        Kind::bytes().or_undefined(),
+                        None,
```

Comment:
This is probably `host`, [ref](https://github.com/vectordotdev/vector/blob/master/src/sources/datadog_agent/mod.rs#L212-L218).
```suggestion
                        Some("host"),
```